lib/mauth/client.rb

```diff
@@ -447,18 +439,27 @@ def get(app_uuid)
               @mauth_client.send(:mauth_service_response_error, response)
             end
           end
-          @cache[app_uuid].security_token
+          @cache[app_uuid]
         end
 
         private
 
+        def security_token_from(response_body)
+          JSON.parse response_body
```

Comment:
Mental note for some future PR on the main branch: use Oj here and anywhere JSON is used since we now bring that dependency in. No need to fix it in this branch, unless you feel like doing so!